app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        person_check = Person.people[person["name"]]
```

Comment:
Accessing `Person.people[person['name']]` assumes that the person with the given name has already been added to the `people` dictionary. Ensure that all persons are added before this line is executed to avoid potential `KeyError`.